testsuite/tests/tool-ocamlopt-stop-after/stop_after_lambda.ml

```diff
@@ -0,0 +1,10 @@
+(* TEST
+* setup-ocamlopt.byte-build-env
+** ocamlopt.byte
+    flags = "-dlambda -stop-after lambda -nopervasives "
+    ocamlopt_byte_exit_status = "0"
+*** check-ocamlopt.byte-output
```

Comment:
I realize now that I was wrong: I thought that you had replaced the ocamlc-only test by an ocamlopt-only test, but in fact you had both an ocamlc-only and an ocamlopt-only test. I'm not sure that you new ocamlc-and-ocamlopt test works -- the CI will tell us. If it does not work, please feel free to revert to the previous state by removing your Fixup? commit, this is good enough, and sorry for the trouble.

Reply:
Before this PR, the compiler already had a separate test directories about ocamlc+stop-after and ocamlopt+stop-after. I placed a test for ocamlopt to separate directory because of this. So, it shouldn't look like I was testing only ocamlopt. You. @gasche, asked from the beginning to put these tests together, but I though that you forgot that there are already distinct directories for two compilers..... Sorry.

Now, I put tests  for `ocamlc -stop-after lambda` and  `ocamlopt -stop-after lambda` both to `testsuite/tests/tool-ocamlc-stop-after`. I don't know why it is better, but you should probably know more than me.

P.S. There is a small difference in the output of `-dlambda` for ocamlc comparately to ocamlopt. I hope it is normal.

Reply:
>  -- the CI will tell us.

@gasche Could you, please, approve workflows?

Reply:
Yes, I had definitely forgotten that there were two different directories and I hadn't understood that you had a test in both. Apologies for the confusion.

I think that, for options that are supported by both ocamlc and ocamlopt, testing the two in the same place (in the same test file) is slightly nicer, but in your case it looks like the test will fail because of this difference. One can have different reference files for the same test, but I feel that you have spent enough time on the tests and that it would be fine to revert to the previous state, with two different tests.

Reply:
The CI indeed failed. I reverted that commit. The CI desires approval once again.

(I'm not sure why it failed, though. On my machine it tested ocamlc, ocamlopt after that, and the output of ocamlopt was concatenated to the output of ocamlc. On my machine it passes, but on CI it looks like somebody flushes the output...)